test/gurka/test_landmarks.cc

```diff
@@ -13,16 +13,58 @@ using namespace valhalla::mjolnir;
 const std::string db_name = "landmarks.db";
 const std::filesystem::path file_path = db_name;
 
+namespace {
+valhalla::gurka::map BuildPBF(const std::string& workdir) {
+  const std::string ascii_map = R"(
+      a-----b-----c---d
+      A   B   C   D   E
+    )";
+
+  const gurka::nodes nodes = {
+      {"A", {{"landmark", "yes"}, {"name", ""}, {"amenity", "university"}}},
+      {"B", {{"landmark", "yes"}, {"name", "hai di lao"}, {"amenity", "restaurant"}}},
+      {"C", {{"landmark", "yes"}, {"name", "ke ji lu"}, {"amenity", ""}}}, // no amenity, log error
+      {"D", {{"landmark", "yes"}, {"name", "wan da"}, {"amenity", "cinema"}}},
+      {"E",
+       {{"landmark", "yes"},
+        {"name", "zhong lou"},
+        {"amenity", "monument"}}}, // not in list, shouldn't be stored
+      // non-landmark nodes
+      {"a", {{"landmark", "no"}, {"name", "gong ce"}, {"amenity", "toilet"}}},
+      {"b", {{"landmark", ""}, {"name", ""}, {"amenity", "trash_can"}}},
+      {"c", {{"name", "hua yuan"}, {"amenity", ""}}},
+      {"d", {{"name", ""}, {"amenity", ""}}},
+  };
+
+  const gurka::ways ways = {
+      {"ab", {}},
+      {"bc", {}},
+      {"cd", {}},
```

Comment:
let's give it at least a `{"highway", "residential"}` or some other highway attribute

Reply:
done